### DISCUSSION TITLE
Ranking by state

### DISCUSSION THREAD

**Discussion by DeadFireFly963** [ppy/osu#29488] [Ideas] [answered]
So this is coming from the osu forums 13 years ago [fourm](https://osu.ppy.sh/community/forums/topics/191135?n=1) which has a high priority among all the others, yet not solved. Could you please add the option to add ur state in your profile so it could be something like #1 Global #1 County #1 State or Providence (Depends on the Country). Along with this, could we add a way for people to change their country on osu if they move countries? Its not common, but it does happen.

**Comment by Jinkku** [comment]
For the changing countries they have implemented that, it's just u have to wait a certain amount of days until u can transfer to that country.

Now for the state, I would love to see that implemented so that I could compete with other people in my state

**Comment by DeadFireFly963** [comment]
That and cities above 100k could have their own leaderboard and not by
state or providence. To make it fair for all players, we should add
clusters of people <50 miles away from you in a nearby tab since people
were talking about that on the forum. If we can't do that, we should get
phone numbers from the players for safety and the first 3 digits in the USA
help who's nearby you. Some information about these codes for peppy is:
(217) Central Illinois, (773) Chicago, Illinois, and (314) Missouri (no
clue where in Missouri). Phone's numbers are a basic way of figuring it out
but people move so there should be a spot in text where you can say what
county and state you moved to. I have a friend who's mom has an Alaska
number but moved to the (217) area.

**Comment by bdach** [comment, marked as answer]
As I said on the forum I do not foresee this happening because people will complain that we're doxxing their location by doing this.
